src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqBurdenEstimateRepository.kt

```diff
@@ -215,6 +217,59 @@ class JooqBurdenEstimateRepository(
         return mapper.mapBurdenEstimateSets(records)
     }
 
+    override fun getBurdenEstimateOutcomesSequence(groupId: String, touchstoneVersionId: String, scenarioId: String, burdenEstimateSetId: Int)
+            : Sequence<BurdenEstimateOutcome>
+    {
+        //check that the burden estimate set exists in the group etc
+        val set = getBurdenEstimateSet(groupId, touchstoneVersionId, scenarioId, burdenEstimateSetId)
+
+        if (set.isStochastic())
+        {
+            throw InvalidOperationError("Cannot get burden estimate data for stochastic burden estimate sets")
+        }
+
+        return  dsl.select(
+                    DISEASE.ID,
+                    BURDEN_ESTIMATE.YEAR,
+                    BURDEN_ESTIMATE.AGE,
+                    COUNTRY.ID,
+                    COUNTRY.NAME,
+                    BURDEN_OUTCOME.CODE,
+                    BURDEN_ESTIMATE.VALUE
+                )
+                .fromJoinPath(BURDEN_ESTIMATE_SET, BURDEN_ESTIMATE)
+                .join(RESPONSIBILITY)
+                .on(BURDEN_ESTIMATE_SET.RESPONSIBILITY.eq(RESPONSIBILITY.ID))
+                .joinPath(RESPONSIBILITY, SCENARIO, SCENARIO_DESCRIPTION, DISEASE)
+                .join(COUNTRY)
+                .on(BURDEN_ESTIMATE.COUNTRY.eq(COUNTRY.NID))
+                .joinPath(BURDEN_ESTIMATE, BURDEN_OUTCOME)
+                .where(BURDEN_ESTIMATE_SET.ID.eq(burdenEstimateSetId))
+                .orderBy(
```

Comment:
Not sure if this ordering is best or desirable. It makes testing easier!

Reply:
Seems reasonable enough!